pyogrio/geopandas.py

```diff
@@ -37,33 +39,56 @@ def _stringify_path(path):
     return path
 
 
-def _try_parse_datetime(ser):
+def _try_parse_datetime(ser, datetimes="UTC"):
     import pandas as pd  # only called when pandas is known to be installed
+    from pandas.api.types import is_string_dtype
+
+    datetimes = datetimes.upper()
+    datetime_kwargs = {}
+    if datetimes == "STRING":
+        if not is_string_dtype(ser.dtype):
+            res = ser.astype("string").str.replace(" ", "T")
+            return res
+        if __gdal_version__ < (3, 7, 0):
+            # GDAL < 3.7 doesn't return datetimes in ISO8601 format, so fix that
+            return ser.str.replace(" ", "T").str.replace("/", "-")
+        return ser
+    elif datetimes == "UTC":
+        datetime_kwargs["utc"] = True
```

Comment:
So by setting this always in the default case, that means we actually no longer preserve tz-naive data as being tz-naive? Since GDAL has that concept somewhat (and depending on the file format), I would prefer that keep that roundtrip functionality.

Reply:
It's a bit annoying that pandas does not give an easy way to get this, and you essentially need to do this try/except calling `to_datetime` a second time ..

Reply:
Sorry, I reinstated the "old" behaviour incorrectly... should be fixed now.